apps/wpcom-block-editor/src/wpcom/features/redirect-onboarding-user-after-publishing-post.js

```diff
@@ -1,15 +1,17 @@
 import { dispatch, select, subscribe } from '@wordpress/data';
-import domReady from '@wordpress/dom-ready';
 import { getQueryArg } from '@wordpress/url';
+import useSiteIntent from './use-site-intent';
 
 const START_WRITING_FLOW = 'start-writing';
 
-export function redirectOnboardingUserAfterPublishingPost() {
-	const isStartWritingFlow = getQueryArg( window.location.search, START_WRITING_FLOW ) === 'true';
+export function RedirectOnboardingUserAfterPublishingPost() {
+	const { intent } = useSiteIntent();
+	console.log( 'intent', intent );
```

Comment:
This is returning an empty string, at the moment.

Reply:
Fixed in https://github.com/Automattic/wp-calypso/pull/76740/commits/e6b680cd4b36dc327b629e1404e122bbbf524b9c